src/m365/teams/commands/report/report-directroutingcalls.ts

```diff
@@ -89,7 +89,7 @@ class TeamsReportDirectroutingcallsCommand extends GraphCommand {
     try {
       const res: { value: any[] } = await request.get<{ value: any[] }>(requestOptions);
       logger.log(res);
-    } 
+    }
```

Comment:
Let's revert this change

Reply:
Can't, VS automatic removes trailing spaces